samples/net/cloud/aws_iot_mqtt/src/tls_config/user-tls.h

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2023 Lucas Dietrich <ld.adecy@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define MBEDTLS_AES_ROM_TABLES
+
+#define MBEDTLS_HAVE_TIME
+#define MBEDTLS_HAVE_TIME_DATE
+#define MBEDTLS_PLATFORM_TIME_ALT
```

Comment:
Is this really needed? The sample doesn't seem to be using `mbedtls_platform_set_time()`